Extensions/bezier/bezier.lisp

```diff
@@ -844,25 +844,110 @@ second curve point, yielding (200 50)."
 
 ;;; Fallback method (suitable for CLX)
 
-(defmethod medium-draw-bezier-design* (medium design)
+(defmethod medium-draw-bezier-design* (medium design options)
   (render-through-pixmap design medium))
 
 
+;;; The real CLX method
+
+(defun my-compute-rgb-image-pixmap (drawable image)
+  (let* ((width (climi::image-width image))
+         (height (climi::image-height image))
+         (depth (xlib:drawable-depth drawable))
+         (im (clim-clx::image-to-ximage-for-drawable drawable image)))
+    (setf width (max width 1))
+    (setf height (max height 1))
```

Comment:
if image width/height is lower than 1 (i.e 0) wouldn't it make sense to just skip whoe computation and skip putting pixmap (and invoking whole xlib machinery here)?